addons/website_slides/models/slide_slide.py

```diff
@@ -167,6 +143,7 @@ class Slide(models.Model):
     embed_code = fields.Html('Embed Code', readonly=True, compute='_compute_embed_code', sanitize=False)
     # views
     embedcount_ids = fields.One2many('slide.embed', 'slide_id', string="Embed Count")
```

Comment:
I know I am that annoying bguy ut ... why not renaming this "embedcount_ids" :D ? Ugly naming is ugly, making sad reviewer sad.